### ISSUE TITLE
Import rig bug in 15.02, hangs

### ISSUE THREAD

**Issue by Nico-Duduf** [Duduf-dev/Duik#182]

**Comment by stevygee** [comment]
Hello,
I think I'm also having this problem. Does it always happen or is there a workaround? Thanks.

**Comment by Nico-Duduf** [comment]
I'm working on a fix, it will be available with lots of other bugfixes in a few days !

**Comment by Nico-Duduf** [comment]
fixed (available in 15.03 in a few days)

**Comment by stevygee** [comment]
Thanks a lot! I was being impatient and already tried it out with the changes you made. Unfortunately it still hangs at "updating expressions" with a rig I created using 15.02. I would be willing to help you debug this, is there some kind of log file that I can send you?

**Comment by Nico-Duduf** [comment]
Updating expressions can be a very long process, and After Effects in this case seems to be frozen, and the progress bar too, but the script should continue to run and you'll have to wait a bit until it processes.
Is that what's happening ?

If when "updating expressions" After Effects isn't frozen, and you can use it as usual, this means the script has ended and may have failed to import the composition. In this case, what you can do is enable javascript debugging in the preferences of After Effects, and post here a screenshot of the debugger which should be opened when the importation fails.

Thanks for your help!

**Comment by Nico-Duduf** [comment]
(Maybe you could try with a very simple rig in an empty project too, that should make the importation quicker)

**Comment by Nico-Duduf** [comment]
I found a bug when duplicating comps and updating expressions, working on it

**Comment by Nico-Duduf** [comment]
fixed

**Comment by stevygee** [comment]
I can confirm that importing works now! Thanks a lot for your quick response.

So far I only noticed one small issue: On my characters (that don't have necks) the importer introduces a new Zero for the head, to which the head controller is parented. The additional offset causes the head controller to be in the wrong place and this messes with the IK spine. But once I set the position of "Zero head" to [0,0] all is well.
